test/oli_web/live/community_live_test.exs

```diff
@@ -80,15 +80,34 @@ defmodule OliWeb.CommunityLiveTest do
       assert has_element?(view, "a[href=\"#{@live_view_index_route}/new\"]")
     end
 
-    test "lists all communities", %{conn: conn} do
+    test "lists only active communities", %{conn: conn} do
       c1 = insert(:community)
       c2 = insert(:community)
```

Comment:
I would remove c2 (I think it's ok to have one active, one not) since c2 isn't adding anything here.